content/docs/payment-providers/quickpay/package.yml

```diff
@@ -0,0 +1,17 @@
+id: Vendr.PaymentProviders.QuickPay
+name: QuickPay
+logo: quickpay.png
+type: PaymentProvider
+supportedBy: core
```

Comment:
I think quickpay will become a core provider so this is fine as is for now, but just a not that for contrib packages, this should be set to `community`

Reply:
Okay, anywhere it is is possible to see accepted values for this? Maybe a comment in readme?

Reply:
Yea, I need to document contributing to the docs so will need to add this. The 2 supported values atm are `core` or `community`